### PR TITLE
Attribute and enum documentation cleanup

### PR DESCRIPTION
**Pull request by texasaggie97**
[X] This contribution adheres to [CONTRIBUTING.md]
### What does this Pull Request accomplish?
* Makes the documentation metadata consistent among all the types:
  * functions
  * attributes
  * enums
* Allows similar formatting regardless of type
  * Gives consistency among the documentation pages

### Why should this Pull Request be merged?
* Clean up

### What testing has been done?
* Travis & system test

### CODE REVIEW

File: build/templates/attributes.rst.mako [hunk 1]
```diff
@@ -28,21 +28,21 @@ table_contents = [
...
-   The following table lists the characteristics of this property.
+    The following table lists the characteristics of this property.
 
-   ${helper.get_indented_docstring_snippet(table, indent=3)}
+    ${helper.get_indented_docstring_snippet(table, indent=4)}
 
-   .. tip:: 
-      This attribute corresponds to the following LabVIEW Property or C Attribute:
+    .. tip:: 
+        This attribute corresponds to the following LabVIEW Property or C Attribute:
```
Comment (marcoskirsch) [inline build/templates/attributes.rst.mako]: The proper term for how we expose these in Python is "properties". I think we should use the right term in documentation.

File: docs/nimodinst/attributes.rst [hunk 2]
```diff
@@ -5,209 +5,209 @@ NI-ModInst Attributes
...
+    +----------------+---------+
+    | Channel Based  | False   |
+    +----------------+---------+
+    | Resettable     | No      |
+    +----------------+---------+
 
-   .. tip:: 
-      This attribute corresponds to the following LabVIEW Property or C Attribute:
+    .. tip:: 
+        This attribute corresponds to the following LabVIEW Property or C Attribute:
```
Comment (marcoskirsch) [inline docs/nimodinst/attributes.rst]: here it says "attribute"

File: generated/nidmm/enums.py [hunk 3]
```diff
@@ -5,632 +5,212 @@
 
 class ADCCalibration(Enum):
     AUTO = -1
-    '''
```
Comment (marcoskirsch) [inline generated/nidmm/enums.py]: Why remove this documentation?

File: generated/nidmm/session.py [hunk 4]
```diff
@@ -555,17 +683,17 @@ class Session(object):
     temp_rtd_a = AttributeViReal64(1150121)
     '''
     Specifies the Callendar-Van Dusen A coefficient for RTD scaling when the
-    **RTD Type property** is set to Custom.
+    **RTD Type property**
```
Comment (marcoskirsch) [inline generated/nidmm/session.py]: It seems like part of the documentation is getting clipped.

File: generated/nidmm/session.py [hunk 5]
```diff
@@ -1233,7 +1348,7 @@ def configure_multi_point(self, trigger_count, sample_count, sample_trigger, sam
     def configure_offset_comp_ohms(self, offset_comp_ohms):
         '''configure_offset_comp_ohms
 
-        For NI 4080/4081/4082 and NI 4070/4071/4072, allows the DMM to
+For NI 4080/4081/4082 and NI 4070/4071/4072, allows the DMM to
```
Comment (marcoskirsch) [inline generated/nidmm/session.py]: Is this desired, for the line to begin with no indentation?

File: src/nidmm/metadata/enums.py [hunk 6]
```diff
@@ -1,34 +1,39 @@
+
 # -*- coding: utf-8 -*-
 # This file is code generated, do not make changes here
 #  If the generated information is not correct for python
-#  changes can be made in attributes_addon.py and they will be
+#  changes can be made in functions_addon.py and they will be
```
Comment (marcoskirsch) [inline src/nidmm/metadata/enums.py]: not enums_addon.py?

File: build/templates/attributes.rst.mako (hunk 1 above)
Comment (texasaggie97) [inline build/templates/attributes.rst.mako]: Changed to property

File: docs/nimodinst/attributes.rst (hunk 2 above)
Comment (texasaggie97) [inline docs/nimodinst/attributes.rst]: Changed to property in template

File: docs/nimodinst/attributes.rst [hunk 7]
```diff
@@ -5,209 +5,209 @@ NI-ModInst Attributes
...
-        - C Attribute: **NIMODINST_ATTR_BUS_NUMBER**
+            - C Attribute: **NIMODINST_ATTR_BUS_NUMBER**
 
 .. py:attribute:: chassis_number
 
-   The number of the chassis in which the device is installed. This attribute can only be queried for PXI devices installed in a chassis that has been properly identified in MAX.
+    
 
-   The following table lists the characteristics of this property.
+    The following table lists the characteristics of this property.
```
Comment (texasaggie97) [inline docs/nimodinst/attributes.rst]: Changed to property in template

File: generated/nidmm/enums.py (hunk 3 above)
Comment (texasaggie97) [inline generated/nidmm/enums.py]: Wasn't removed on purpose. Added back.

File: generated/nidmm/session.py (hunk 4 above)
Comment (texasaggie97) [inline generated/nidmm/session.py]: Fixed incomplete documentation

File: generated/nidmm/session.py (hunk 5 above)
Comment (texasaggie97) [inline generated/nidmm/session.py]: Fixed indentation

File: src/nidmm/metadata/enums.py (hunk 6 above)
Comment (texasaggie97) [inline src/nidmm/metadata/enums.py]: Changed

**Pull request merged**